runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/fieldwatch/fw001.c

```diff
@@ -0,0 +1,260 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "jvmti_test.h"
+#include <string.h>
+#include <stdio.h>
+#include <errno.h>
+
+jvmtiEnv *globalEnv;
+jfieldID currentFieldBeingWatched;
+const char * testClassName;
+jstring testClassString;
+
+
+    // Find the correct API to increment the number of reports we have seen and return the new value.
+    jint incAndGetNumReports(JNIEnv* jni_env)
+    {
+        jclass clazz = (*jni_env)->FindClass(jni_env, testClassName);
+        if (clazz == NULL)
```

Comment:
rvalues go on the LHS of equality/inequality

Reply:
Addressed this here: https://github.com/eclipse/openj9/pull/6257/commits/c924c857a764969d32dafe29af2f7359c334d150